src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -479,22 +479,22 @@ private static async Task<object> ExecuteHubMethod(ObjectMethodExecutor methodEx
         private void InitializeHub(THub hub, HubConnectionContext connection)
         {
             hub.Clients = new HubCallerClients(_hubContext.Clients, connection.ConnectionId);
-            hub.Context = new DefaultHubCallerContext(connection);
+            hub.Context = connection.HubCallerContext;
```

Comment:
YESSS

Reply:
Why is this exciting?

Reply:
Because this is removing one of our allocations from the hot path.